homeassistant/components/sky_hub/device_tracker.py

```diff
@@ -67,45 +75,11 @@ def _update_info(self):
 
         _LOGGER.info("Scanning")
```

Comment:
I recommend removing this and instead log the result of the scan at debug level below.

Reply:
This is ok for now.
```suggestion
        _LOGGER.debug("Scanning")
```